src/actors/player.gd

```diff
@@ -55,12 +57,28 @@ enum Weapon {
 var action_state = Action.MOVE
 var weapon_state = Weapon.GUN
 
-var slash_frame = "slash2"
-
 onready var anim = $PlayerAnimation
 onready var player_shadow = $PlayerAnimation/PlayerShadow
 onready var health_bar = $Healthbar
 
+## For Boons
+# Idle Heal
+var has_idle_heal_boon = false
+var idle_time_limit = 3
+var idle_heal_amount = 0
+var is_idle = false
```

Comment:
The repetition of `idle` here is making my OO alarms go off that we're missing some abstraction here. Some sort of `idle` object that can encapsulate all these various bits of state.